packages/dds/tree/src/tree/treeTextFormat.ts

```diff
@@ -86,7 +91,7 @@ export function scopeFromKey(key: FieldKey): [FieldScope, LocalFieldKey | Global
  * Derives the scope using the type of `key`.
  */
 export function isGlobalFieldKey(key: FieldKey): key is GlobalFieldKeySymbol {
-    return typeof key === "symbol";
+    return typeof key === "symbol" && symbolIsFieldKey(key);
```

Comment:
this change is unneeded. Its invalid to have a FieldKey thats a symbol thats not a field key symbol. Getting to such a state would require someone to have done an invalid type conversion.

If we do keep this check here, it should be an assert, but that should not be necessary.

Reply:
Valid point. Reverted.